### PR TITLE
Upgrade lodash to 4.17.13

### PR DESCRIPTION
**Pull request by CYBAI**
As @jdalton [said](https://twitter.com/jdalton/status/1148833761301164032), it's time to update lodash so that we can fix [the security vulnerability](https://snyk.io/blog/snyk-research-team-discovers-severe-prototype-pollution-security-vulnerabilities-affecting-all-versions-of-lodash/) reported by Snyk.

| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            |  
| Patch: Bug Fix?          | No
| Major: Breaking Change?  | No
| Minor: New Feature?      | No
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  | Upgrade all `lodash` to `4.17.13`
| License                  | MIT

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6470,6 +6470,11 @@ lodash@^4.13.1, lodash@^4.15.0, lodash@^4.17.10, lodash@^4.17.11, lodash@^4.17.4
   resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.11.tgz#b39ea6229ef607ecd89e2c8df12536891cac9b8d"
```
Comment (JLHwung) [inline yarn.lock]: There will be another duplicate lodash 4.17.11 inside `node_modules`. Could you run
```
yarn-deduplicate --package lodash ./yarn.lock
```
[yarn-deduplicate](https://github.com/atlassian/yarn-deduplicate#limit-packages-to-deduplicate-yarnlock) is a third party tool helps keep dependency flat.

Comment (CYBAI) [reply yarn.lock]: Thanks! I ran it and amended it!

File: yarn.lock [hunk 2]
```diff
@@ -6948,12 +6948,7 @@ node-fetch@^1.7.0:
     encoding "^0.1.11"
     is-stream "^1.0.1"
 
-node-fetch@^2.3.0:
-  version "2.5.0"
-  resolved "https://registry.yarnpkg.com/node-fetch/-/node-fetch-2.5.0.tgz#8028c49fc1191bba56a07adc6e2a954644a48501"
-  integrity sha512-YuZKluhWGJwCcUu4RlZstdAxr8bFfOVHakc1mplwHkk8J+tqM1Y5yraYvIUpeX8aY7+crCwiELJq7Vl0o0LWXw==
-
-node-fetch@^2.5.0:
+node-fetch@^2.3.0, node-fetch@^2.5.0:
```
Comment (JLHwung) [inline yarn.lock]: It seems unrelated to lodash upgrade. But I am happy to leave it as-is.

Comment (CYBAI) [reply yarn.lock]: Hmm, maybe it's caused by running `yarn-deduplicate` 🤔? If others hope this change to be removed, please let me know!
